vrayGolaem/vraygolaem.cpp

```diff
@@ -182,10 +182,10 @@ static ParamBlockDesc2 param_blk(params, STR_DLGTITLE,  0, &vrayGolaemClassDesc,
 	p_default, TRUE,
 	p_ui, TYPE_SINGLECHEKBOX, ED_DISPLAYENABLE,
 	PB_END,
-	pb_display_percent, _T("display_percent"), TYPE_INT, P_RESET_DEFAULT, 0,
-	p_default, 100,
-	p_range, 0, 100, 
-	p_ui, TYPE_SPINNER,  EDITTYPE_POS_INT, ED_DISPLAYPERCENT, ED_DISPLAYPERCENTSPIN, 1,
+	pb_display_percentage, _T("display_percentage"), TYPE_FLOAT, P_RESET_DEFAULT, 0,
```

Comment:
Hi there,

I actually had to change the type of the attribute (from int to float) so I
had to rename the attribute (else it broke compatibility with previous
scenes)
I kept the previous parameter below in the not used attributes:
    // not used anymore but kept for retrocomp
    pb_display_percent, _T(""), TYPE_INT, 0, 0, PB_END,

MaxScripts will indeed break but people will notice that the parameter is
new and has more control
So, ok with this :)


---
[image: Nicolas Chaverou - Golaem Crowd Product Manager - Golaem SA]

+33 (0)2 99 27 21 44
http://www.golaem.com

On Tue, Jul 12, 2016 at 12:57 PM, vkoylazov <notifications@github.com>
wrote:

> In vrayGolaem/vraygolaem.cpp
> <https://github.com/ChaosGroup/vraygolaem_max/pull/13#discussion_r70417570>
> :
>
> > @@ -182,10 +182,10 @@ static ParamBlockDesc2 param_blk(params, STR_DLGTITLE,  0, &vrayGolaemClassDesc,
> >  	p_default, TRUE,
> >  	p_ui, TYPE_SINGLECHEKBOX, ED_DISPLAYENABLE,
> >  	PB_END,
> > -	pb_display_percent, _T("display_percent"), TYPE_INT, P_RESET_DEFAULT, 0,
> > -	p_default, 100,
> > -	p_range, 0, 100,
> > -	p_ui, TYPE_SPINNER,  EDITTYPE_POS_INT, ED_DISPLAYPERCENT, ED_DISPLAYPERCENTSPIN, 1,
> > +	pb_display_percentage, _T("display_percentage"), TYPE_FLOAT, P_RESET_DEFAULT, 0,
>
> Changing the name will break any existing MaxScripts that use this
> parameter. Are you ok with this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ChaosGroup/vraygolaem_max/pull/13/files/1c759ff95b74d05239a073202f2aa64d3a919788#r70417570>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADWXtsye0Cd2I2UBhLt76pCD9dZlUtwgks5qU3MkgaJpZM4JKRcy>
> .
>